tutorials/zh/7_unsupervised_learning_with_graphsage.ipynb

```diff
@@ -29,7 +29,8 @@
    "source": [
     "# Install graphscope package if you are NOT in the Playground\n",
     "\n",
-    "!pip3 install graphscope"
+    "!pip3 install graphscope\n",
+    "!pip3 uninstall -y importlib_metadata  # Address an module conflict issue on colab.google. Remove this line if you are not on colab."
```

Comment:
Why there's a conflict?

Reply:
Some error info like `'posixpath' object has no attribute 'read_text'` in `importlib_metadata.__init__` during `import tensorflow (from import graphlearn)` in our coordinator